tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -47,10 +47,11 @@ struct choose_literal<wchar_t> {
 
 template <class charT, class... Args>
 auto make_testing_format_args(Args&&... vals) {
+    using context = basic_format_context<back_insert_iterator<basic_string<charT>>, charT>;
     if constexpr (is_same_v<charT, wchar_t>) {
-        return make_wformat_args(forward<Args>(vals)...);
+        return make_wformat_args<context>(forward<Args>(vals)...);
     } else {
-        return make_format_args(forward<Args>(vals)...);
+        return make_format_args<context>(forward<Args>(vals)...);
     }

```

Comment:
This entire `if constexpr` should be simply `return make_format_args<context>(forward<Args>(vals)...);`.

Reply:
I thought this is also testing `make_wformat_args`.

Reply:
I don't think it's important that we test that `make_wformat_args` is non-conforming =)

Reply:
Kept `make_wformat_args` because it is not exercised anywhere else in tests and needs fixing (#1853).

Reply:
> I don't think it's important that we test that make_wformat_args is non-conforming

The non-conformance is fixed in #1853. Here we just make non-conformance obvious.

Reply:
This will break after application of #1853. I'd like to keep the two PRs orthogonal to each other, which I think we can best do by (1) changing this code to avoid `make_wformat_args` and (2) adding `make_wformat_args` coverage in #1853.

Reply:
> and (2) adding `make_wformat_args` coverage in #1853.

This goes double given that the result of `make_format_args` is non-conforming since it doesn't have a `std::array` member named `args`, which makes it hard to write test coverage. =(